pkgs/os-specific/darwin/apple-source-releases/removefile/default.nix

```diff
@@ -1,4 +1,4 @@
-{ appleDerivation }:
+{ appleDerivation, headersOnly ? true }:
```

Comment:
Why do you add an unused parameter? This will be removed.

Reply:
The origin purpose is marking them headersOnly and to add `meta.broken = ! headersOnly;`, and Libsystem can explicitly use those headersOnly version later. This is sort of reminder of incomplete packages (we don't compile at all), to avoid others misusing. I will drop them later.

Is there a better way to leave that message? (modify `name` or comment in code only?)